package.json

```diff
@@ -217,7 +217,7 @@
     "ts-loader": "^6.2.1",
     "ts-node": "~10.8.1",
     "tslint": "~6.1.0",
-    "typescript": "~4.0.8",
+    "typescript": "4.1.2",
```

Comment:
is this intentional?

Reply:
yeah some thing was breaking when running the app - let me revert and try and run app to see error message